tests/e2e.test.ts

```diff
@@ -384,5 +385,45 @@ describe('End-to-end tests', () => {
         `)
       })
     })
+    describe('--error-format option', () => {
```

Comment:
You guys think a test is needed to ensure the default format of `error-format` remains `human`-readable?

Reply:
I don't think so, the existing tests already check that, right?